iquip/apps/logger.py

```diff
@@ -125,52 +137,75 @@ class LoggerApp(qiwis.BaseApp):
     Attributes:
         loggerFrame: A frame that shows the logs.
         confirmFrame: A frame that asks whether to clear logs.
-        handler: A handler for adding logs to the loggerFrame. 
+        levelsDict: A dictionary that matches int level to string level.
+        frameHandler: A handler for adding logs to the loggerFrame.
+        fileHandler: A handler for saving logs to file.
     """
 
-    def __init__(self, name: str, parent: Optional[QObject] = None):
-        """Extended."""
+    def __init__(self, name: str, path: str, parent: Optional[QObject] = None):
```

Comment:
I think it is better that the `path` has a default value such as ".".

Reply:
I requested to remove the default value because the "current directory" depends on where the program is executed.

Reply:
Oh, I realize it now, but how about `logFilePath`?
I think `path` is little vague, because there is already `path` argument in `setup.json`.